config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/FlightPotion-4LY2S8QsSw-JdMGrF8FUsQ==.json

```diff
@@ -0,0 +1,124 @@
+{
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 1598
+    }
+  },
+  "properties:10": {
+    "betterquesting:10": {
+      "autoClaim:1": 0,
+      "desc:8": "Tired of jetpacks? Want normal flight with no strings attached? Look no further!\n\nUsing your brand new MV Assembler and a Chicken Trophy, you can craft a Potion of Flight, giving you 7 minutes of Creative flight with 8 uses before you have to make a new one.\n\n[note]Chicken Trophies are a rare drop from killing chickens.[/note]",
```

Comment:
As a way to encourage players to explore Blood Magic, please also include the following information as part of the text:
- You can craft your own Flight Potion Flasks with a flight time of up to 19 minutes.
- You can use any Filling Agent and any Blood Orb in an Alchemic Chemistry Set to refill these potions.